script/migrations/01-deploy_credits.sol

```diff
@@ -11,7 +11,7 @@ import "forge-std/Script.sol";
 import "forge-std/console.sol";
 
 contract KintoMigration1DeployScript is Create2Helper, ArtifactsReader {
-    using ECDSAUpgradeable for bytes32;
+    using MessageHashUtils for bytes32;
```

Comment:
ECDSA has been split into ECDSA and this new library (more info [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/CHANGELOG.md?plain=1#L152))